build-utils/webpack.dev.js

```diff
@@ -45,5 +45,13 @@ module.exports = {
     port: 3000,
     historyApiFallback: true,
   },
+  resolve: {
+    alias: {
+      assets: path.resolve(__dirname, '..', 'src/assets/'),
+      pages: path.resolve(__dirname, '..', 'src/pages/'),
+      components: path.resolve(__dirname, '..', 'src/components/'),
+      sass: path.resolve(__dirname, '..', 'src/assets/styles/sass/'),
+    },
+  },
```

Comment:
This is for absolute imports, need to install with `eslint-import-resolver-webpack`